releasenotes/notes/accept-extra-config-files-f0ab26a4c6bc78c0.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    The core Agent now supports multiple configuration files in addition to the main datadog.yaml file. 
+    The -E flag can be used with the ``datadog-agent run`` command to specify multiple configuration paths.
+    Configuration files are merged sequentially. If a key already exists, subsequent occurrences are ignored.
```

Comment:
what do you mean by `If a key already exists, subsequent occurrences are ignored.` ?

Reply:
I guess we are missing `of the same key`

`If a key already exists, subsequent occurrences of the same key are ignored.`

Reply:
@GustavoCaso is right ! I added some details in 253d5813e8de843aaa0af81f99f508288aaf45d6.